### PR TITLE
Add styled components

### PR DESCRIPTION
**Pull request by michael-albrecht-1**
- styled component and types dependencies added
- sharedStyle/styles.ts created with some reusable styles
- styled file was attached to each component and the old design is restored expect Beers because we'll change interface soon

### CODE REVIEW

File: src/components/Beers/Beers.styled.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  StyledInputGroup, 
+  StyledButtonGroup, 
+} from '../../sharedStyle/styles';
+
+export const StyledBeersContent = styled.div`
+display: flex;
+flex-direction: column;
+color: ${white};
+`
+export const StyledTitle = styled(StyledH1)``;
```
Comment (SimonBoeuf) [inline src/components/Beers/Beers.styled.ts]: Maybe it's a good idea to export a `StyledTitle` that extends `StyledH1` but I'm not sure about doing by default `styled(StyledH1)`` `, maybe just `export const StyledTitle = StyledH1;` is enough as long as we don't want to customize it. I'm not sure about this though, What do you think ?

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+body {
```
Comment (SimonBoeuf) [inline src/index.css:108]: usually it's nice to have a reset css here, it saves a lot of debugging and cross browser implementation issues

**Review by SimonBoeuf — APPROVED** [review]

File: src/components/Beers/Beers.styled.ts (hunk 1 above)
Comment (michael-albrecht-1) [reply src/components/Beers/Beers.styled.ts]: I agree, i just didnt had the idea ;) Code is updated (see last commit)

File: src/index.css (hunk 2 above)
Comment (michael-albrecht-1) [reply src/index.css:108]: Meyer Reset added in the last commit

**Review by SimonBoeuf — APPROVED** [review]

**Pull request merged**